src/popupmenu.c

```diff
@@ -416,6 +416,114 @@ pum_under_menu(int row, int col, int only_redrawing)
 	    && col < pum_col + pum_width + pum_scrollbar;
 }
 
+/*
+ * displays text on the popup menu with specific attributes.
+ */
+    static void
+pum_screen_put_with_attr(int row, int col, char_u *text, int textlen, int attr)
+{
+    int		i;
+    int		leader_len;
+    int		char_len;
+    int		cells;
+    int		new_attr;
+    char_u	*rt_leader = NULL;
+    char_u	*match_leader = NULL;
+    char_u	*ptr = text;
+    garray_T	*ga = NULL;
+    char_u	*leader = ins_compl_leader();
+    unsigned	cur_cot_flags = get_cot_flags();
+    int		in_fuzzy = (cur_cot_flags & COT_FUZZY) != 0;
```

Comment:
```suggestion
    int		in_fuzzy = (get_cot_flags() & COT_FUZZY) != 0;
```
and then get rid of the cur_cot_flags please